proton-j/src/main/java/org/apache/qpid/proton/reactor/impl/TaskImpl.java

```diff
@@ -29,10 +29,12 @@
 import org.apache.qpid.proton.reactor.Task;
 
 public class TaskImpl implements Task, Comparable<TaskImpl> {
+
+    private static final AtomicInteger count = new AtomicInteger();
```

Comment:
This seems like an actual bug, so it deserves a JIRA. Variable name syntax is now wrong.

Reply:
Ok, I will create a Jira and send the PR. Will rename to `COUNT`